docs/content/docs/tutorials/lambda-oci-example.md

```diff
@@ -0,0 +1,184 @@
+exports.handler = async (event) => {
+    const response = {
+        statusCode: 200
+        body: JSON.stringify('Hello from Lambda!')
+    };
+    return response;
+};
+EOF
+```
+
+## Create and Build a Docker Image
+Create a Dockerfile that will be used to build the image for our Lambda function:
+
+```bash
+cat <<EOF > Dockerfile
+FROM public.ecr.aws/lambda/nodejs:14
+
+COPY app.js package.json ./
+
+RUN npm install
+
+CMD [ "app.handler" ]
+EOF
+```
+Build the Docker image in your local environment. You will need to install dependencies using `npm`:
+
+```shell
+npm init -y
+docker build -t hello-world .
+```
+## Publish the Docker image to ECR
+Publish the Docker image to an ECR repository. It's a requirement for container images to be published to the ECR repository to run Lambda OCI image functions.
+
+```shell
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export AWS_REGION=us-west-2
+
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com
+aws ecr create-repository --repository-name hello-world --image-scanning-configuration scanOnPush=true --image-tag-mutability MUTABLE
+docker tag  "hello-world:latest ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/hello-world:latest"
+docker push "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/hello-world:latest"
+```
+
+## Deploy the Lambda OCI function using the ACK Lambda controller
+The following example creates a manifest that contains the Lambda OCI function. It then uses `kubectl` to create the resource in Kubernetes:
+
+```shell
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export IMAGE_URI="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/hello-world:latest "
+export FUNCTION_NAME="lambda-oci-ack"
+export LAMBDA_ROLE="arn:aws:iam::${AWS_ACCOUNT_ID}:role/lambda_basic_execution"
+
+read -r -d '' LAMBDA_MANIFEST <<EOF
+apiVersion: lambda.services.k8s.aws/v1alpha1
+kind: Function
+metadata:
+ name: $FUNCTION_NAME
+ annotations:
+   services.k8s.aws/region: $AWS_REGION
+spec:
+ name: $FUNCTION_NAME
+ packageType: Image
+ code:
+     imageURI: $IMAGE_URI
+ role: $LAMBDA_ROLE
+ description: function created by ACK lambda-controller e2e tests
+EOF
+
+echo "${LAMBDA_MANIFEST}" > function.yaml
+
+kubectl create -f function.yaml
+```
+You should get a confirmation that the function was created successfully.
+
+```
+function.lambda.services.k8s.aws/lambda-oci-ack created
+```
+To get details about the Lambda function, run the following.
+
+```bash
+kubectl describe "function/${FUNCTION_NAME}"
+```
+
+## Invoke the Lambda OCI Function
+After you have verified that the Lambda OCI function is deployed correctly, you can invoke the function through the [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/lambda/index.html).
+
+```bash
+aws lambda invoke --function-name ${FUNCTION_NAME} response.json
```

Comment:
tiny nit: you can also use `aws lambda invoke --function-name --region us-west-2 demo-function-list-items /dev/stdout | jq`